Update ACS Version

stuft2 opened the pull request.

GaryGSC reviewed (approved):
Simple enough.

[Release notes for `terraform-aws-acs-info` v3.2.0](https://github.com/byu-oit/terraform-aws-acs-info/releases/tag/v3.2.0)

stuft2 commented:
@blakemorgan @GaryGSC I updated the ci to test versions 0.12 -> 1.0 of terraform so we get a more accurate view of what versions of terraform we support. I also updated the readme to show the new version of this module that we'll publish when this PR is merged.

GaryGSC commented on .github/workflows/ci.yml:
```diff
@@ -15,15 +15,33 @@ jobs:
+                "tf_working_dir":"./examples/ci-0_13",
+                "aws_key_name":"byu_oit_terraform_dev_key",
+                "aws_secret_name":"byu_oit_terraform_dev_secret"
+              }
```
```suggestion
              },
```

GaryGSC reviewed (approved):
Thanks for expanding the CI matrix.

I think this is a strict improvement over what we had previously, so even with the failing CI, this LGTM.
